packages/frontend-2/components/project/model-page/versions/Card.vue

```diff
@@ -40,7 +40,11 @@
               <PreviewImage :preview-url="version.previewUrl" />
             </NuxtLink>
             <div
-              v-if="!isPendingVersionFragment(version) && version.automationsStatus"
+              v-if="
```

Comment:
@Mikehrn I already hid the donut in a couple of places, cause it was ruining a screen recording im doing for marketing

Reply:
Nice, thanks!